src/MongoDB.Driver.Core/Core/Misc/StreamExtensionMethods.cs

```diff
@@ -24,6 +24,18 @@ namespace MongoDB.Driver.Core.Misc
     internal static class StreamExtensionMethods
     {
         // static methods
+        public static void EfficientCopyTo(this Stream input, Stream output)
+        {
+            if (input is IStreamEfficientCopyTo efficientCopyTo)
```

Comment:
Recommended name change to efficientCopyToStream

Or *something* with the word *Stream* in it.